tests/ci/clickhouse_helper.py

```diff
@@ -96,7 +110,7 @@ def _select_and_get_json_each_row(self, db, query):
                     logging.warning("Reponse text %s", response.text)
                 time.sleep(0.1 * i)
 
-        raise Exception("Cannot insert data into clickhouse")
+        raise Exception("Cannot fetch data from clickhouse")
 
     def select_json_each_row(self, db, query):
```

Comment:
It looks like `select` is not used outside the file, so I don't touch it. `mark_flaky_tests` already process exceptions